ocean_provider/validation/provider_requests.py

```diff
@@ -21,15 +26,30 @@ class CustomJsonRequest(JsonRequest):
     def __init__(self, request=None):
         request = request or flask_request
         request = get_request_data(request)
-        self._validator = CustomValidator(
-            rules=self.rules(),
-            messages={
-                "signature.signature": "Invalid signature provided.",
-                "signature.download_signature": "Invalid signature provided.",
-            },
-            request=request,
+        class_name = self.__class__.__name__
+        self._validators = list()
+        action_mapping = RBACValidator.get_action_mapping()
+        if os.getenv("RBAC_SERVER_URL") and class_name in action_mapping.keys():
+            self._validators.append(
+                RBACValidator(request_name=class_name, request=request)
+            )
+        self._validators.append(
+            CustomValidator(
+                rules=self.rules(),
+                messages={
+                    "signature.signature": "Invalid signature provided.",
+                    "signature.download_signature": "Invalid signature provided.",
+                },
+                request=request,
+            )
         )
 
+    def validate(self):
+        for validator in self._validators:
+            if validator.fails():
+                raise ValidationException(validator.messages())
```

Comment:
please implement the messages function in the validator :)
a simple statement to return "RBAC Validation failed" would be enough.